### PR TITLE
Trim duplication in docker compose guide

### PR DESCRIPTION
**Pull request by Hoverbear**
### What is changed?

We were replicating a lot of content from the official Docker Compose guide. and this guide had some out of date information (eg we have docker swarm now!)

### Any related PRs or issues?

#81

### CODE REVIEW

File: content/docs/3.0/tasks/deploy/docker-compose.md [hunk 1]
```diff
@@ -1,165 +1,20 @@
...
-    OK
-    127.0.0.1:7369> get a
-    "1"
-    ```
-
-## What's next?
-
-+ If you want to try the Go client, see [Try Two Types of APIs](../clients/go-client-api.md). You need to build your docker image and add it to the docker compose config file like above YCSB or Titan does. 
-+ If you want to run a full cluster with monitor support, please follow the [tidb-docker-compose guide](https://github.com/pingcap/tidb-docker-compose/blob/master/README.md), comment the `tidb` and `tispark` sections out in the [values.yaml](https://github.com/pingcap/tidb-docker-compose/blob/master/compose/values.yaml), generate the new docker compose config, then add your own binary image and run it.
+We recommend using the [**Docker Swarm option**](https://github.com/pingcap/tidb-docker-compose#docker-swarm) option for using TiKV. The [Docker Compose](https://github.com/pingcap/tidb-docker-compose#quick-start) option requires some [manual configuration to interact with TiKV directly](https://github.com/pingcap/tidb-docker-compose#host-network-mode-linux) on non-Linux systems.
```
Comment (dcalvin) [inline content/docs/3.0/tasks/deploy/docker-compose.md]:
```suggestion
For non-Linux systems, the [Docker Compose](https://github.com/pingcap/tidb-docker-compose#quick-start) option requires some [manual configuration to interact with TiKV directly](https://github.com/pingcap/tidb-docker-compose#host-network-mode-linux). In this case, we recommend using the [**Docker Swarm**](https://github.com/pingcap/tidb-docker-compose#docker-swarm) option for deploying TiKV.
```

File: content/docs/3.0/tasks/deploy/docker-compose.md [hunk 2]
```diff
@@ -1,165 +1,20 @@
...
 
-This guide describes how to quickly deploy a TiKV testing cluster using [Docker Compose](https://github.com/pingcap/tidb-docker-compose/) on a single machine. Currently, this installation method only supports the Linux system.
+This guide describes how to deploy a single-node TiKV cluster using Docker Compose, or a multi-node TiKV cluster using Docker Swarm.
 
 {{< warning >}}
 **Warning:** Do not use Docker Compose to deploy the TiKV cluster in the production environment. For production, [use Ansible to deploy the TiKV cluster](../ansible).
 {{< /warning >}}
 
-## Prerequisites
+PingCAP (the original authors of TiKV, and a maintaining organization of TiKV), develops and maintains the Apache 2 licensed [TiDB docker-compose](https://github.com/pingcap/tidb-docker-compose). TiDB docker-compose is a collection of Docker Compose files which enable enable you to quickly 'test drive' TiKV as well as TiDB, TiSpark, and the monitoring tools.
```
Comment (dcalvin) [inline content/docs/3.0/tasks/deploy/docker-compose.md]:
```suggestion
PingCAP (the original authors of TiKV, and a maintaining organization of TiKV), develops and maintains the Apache 2 licensed [TiDB docker-compose](https://github.com/pingcap/tidb-docker-compose). TiDB docker-compose is a collection of Docker Compose files which enable you to quickly 'test drive' TiKV as well as TiDB, TiSpark, and the monitoring tools.
```

File: content/docs/3.0/tasks/deploy/docker-compose.md [hunk 3]
```diff
@@ -1,165 +1,20 @@
...
 
-Make sure you have installed the following items on your machine:
+TiDB docker-compose is compatible with Linux as well as Mac and Windows though [Docker Desktop](https://www.docker.com/products/docker-desktop).
```
Comment (dcalvin) [inline content/docs/3.0/tasks/deploy/docker-compose.md]:
```suggestion
TiDB docker-compose is compatible with Linux as well as Mac and Windows through [Docker Desktop](https://www.docker.com/products/docker-desktop).
```

File: content/docs/3.0/tasks/deploy/docker-compose.md (hunk 1 above)
Comment (dcalvin) [reply content/docs/3.0/tasks/deploy/docker-compose.md]: Adjusted the sequence to make the recommendation sound more convincing

**Comment by dcalvin** [comment]
Some overall comment - do you think we need a separate doc here with the actual content removed or linked to somewhere else?

**Comment by Hoverbear** [comment]
@dcalvin I'm not sure. I think it aids discoverability as it becomes a menu item and this page lets us later go ahead and add jump links to useful tutorials, guides, or other pages.

File: content/docs/3.0/tasks/deploy/docker-compose.md (hunk 1 above)
Comment (Hoverbear) [reply content/docs/3.0/tasks/deploy/docker-compose.md]: Hmm, I'd prefer to recommend Swarm mode in all cases as I don't really want to have to support the Docker Compose method. :( It's technology that is on the way out (Swarm is replacing it) and has a lot of dependencies.

Comment (Hoverbear) [reply content/docs/3.0/tasks/deploy/docker-compose.md]: I tried to clean this up.

**Comment by dcalvin** [comment]
> @dcalvin I'm not sure. I think it aids discoverability as it becomes a menu item and this page lets us later go ahead and add jump links to useful tutorials, guides, or other pages.

Sounds fine. Let's merge this for now.

**Pull request merged**